build/docker/Release.dockerfile

```diff
@@ -48,6 +62,5 @@ RUN set -eux && \
     apk del gnupg openssl && \
     rm -rf /root/.gnupg
 
-USER vault
-
-ENTRYPOINT ["/bin/vault-k8s"]
+USER ${NAME}
```

Comment:
This should be `vault`, not `vault-k8s`.

Reply:
Thanks, updated.